pkg/collect/copy_from_host.go

```diff
@@ -125,7 +126,7 @@ func (c *CollectCopyFromHost) Collect(progressChan chan<- interface{}) (Collecto
 	}
 }
 
-func copyFromHostCreateDaemonSet(ctx context.Context, client kubernetes.Interface, collector *troubleshootv1beta2.CopyFromHost, hostPath string, namespace string, generateName string, labels map[string]string) (name string, cleanup func(), err error) {
+func copyFromHostCreateDaemonSet(ctx context.Context, client kubernetes.Interface, collector *troubleshootv1beta2.CopyFromHost, retryFailedMount bool, hostPath string, namespace string, generateName string, labels map[string]string) (name string, cleanup func(), err error) {
```

Comment:
Nit pick: Instead of creating `retryFailedMount` new parameter, just make these private functions as class methods then you can reference internal fields like `c.retryFailedMount`. You might clean up other parameters out but you don't have to in this PR

```suggestion
func (c *CollectCopyFromHost) copyFromHostCreateDaemonSet(ctx context.Context, client kubernetes.Interface, collector *troubleshootv1beta2.CopyFromHost, hostPath string, namespace string, generateName string, labels map[string]string) (name string, cleanup func(), err error) {
```

Reply:
nice, no need for new parameters for private functions. PR has been updated